control/controlclient/status.go

```diff
@@ -75,9 +74,8 @@ type Status struct {
 	// package, but we have some automated tests elsewhere that need to
 	// use them. Please don't use these fields.
 	// TODO(apenwarr): Unexport or remove these.
-	State    State
-	Persist  *persist.Persist  // locally persisted configuration
-	Hostinfo *tailcfg.Hostinfo // current Hostinfo data
```

Comment:
Is `Hostinfo` just not used here? It's not immediately obvious this is related

Reply:
It isn't being used anywhere. See commit message